src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Shipped.txt

```diff
@@ -188,7 +188,6 @@ NuGet.PackageManagement.NuGetPackageManager.PreviewUpdatePackagesAsync(string pa
 NuGet.PackageManagement.NuGetPackageManager.RestorePackageAsync(NuGet.Packaging.Core.PackageIdentity packageIdentity, NuGet.ProjectManagement.INuGetProjectContext nuGetProjectContext, NuGet.Protocol.Core.Types.PackageDownloadContext downloadContext, System.Collections.Generic.IEnumerable<NuGet.Protocol.Core.Types.SourceRepository> sourceRepositories, System.Threading.CancellationToken token) -> System.Threading.Tasks.Task<bool>
 NuGet.PackageManagement.NuGetPackageManager.UninstallPackageAsync(NuGet.ProjectManagement.NuGetProject nuGetProject, string packageId, NuGet.PackageManagement.UninstallationContext uninstallationContext, NuGet.ProjectManagement.INuGetProjectContext nuGetProjectContext, System.Threading.CancellationToken token) -> System.Threading.Tasks.Task
 NuGet.PackageManagement.NuGetProjectAction
-NuGet.PackageManagement.NuGetProjectAction.NuGetProjectAction(NuGet.Packaging.Core.PackageIdentity packageIdentity, NuGet.PackageManagement.NuGetProjectActionType nuGetProjectActionType, NuGet.ProjectManagement.NuGetProject project, NuGet.Protocol.Core.Types.SourceRepository sourceRepository = null) -> void
```

Comment:
If you haven't seen it already: https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md

`NuGet.PackageManagement` is a special case, but is it difficult to keep the old method, add an overload, and make the old method call the overload with a default value for any new parameters?